ur_controllers/src/force_torque_sensor_controller.cpp

```diff
@@ -0,0 +1,180 @@
+//
+// Created by livanov93 on 12/12/20.
+//
+
+#include "ur_controllers/force_torque_sensor_controller.h"
+
+namespace ur_controllers
+{
+ForceTorqueStateController::ForceTorqueStateController() : controller_interface::ControllerInterface()
+{
+}
+
+controller_interface::return_type ForceTorqueStateController::init(const std::string& controller_name)
+{
+  auto ret = ControllerInterface::init(controller_name);
+  if (ret != controller_interface::return_type::SUCCESS)
+  {
+    return ret;
+  }
+
+  return controller_interface::return_type::SUCCESS;
+}
+
+controller_interface::InterfaceConfiguration ForceTorqueStateController::command_interface_configuration() const
+{
+  controller_interface::InterfaceConfiguration config;
+  config.type = controller_interface::interface_configuration_type::NONE;
+  return config;
+}
+
+controller_interface::InterfaceConfiguration ForceTorqueStateController::state_interface_configuration() const
+{
+  controller_interface::InterfaceConfiguration config;
+  config.type = controller_interface::interface_configuration_type::INDIVIDUAL;
+  // TODO solve this hardcoded naming
```

Comment:
Use simply parameter as done for [`ForwardCommandController`](https://github.com/ros-controls/ros2_controllers/blob/master/forward_command_controller/src/forward_command_controller.cpp).

Otherwise at least correct for `ament_cpplint`:
```suggestion
  // TODO(anyone): solve this hardcoded naming
```